aldryn_translation_tools/views.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from django.http import (
+    Http404,
+    HttpResponseRedirect,
+    HttpResponsePermanentRedirect,
+)
+
```

Comment:
It is common to separate direct imports from django "from django... import ..." statements from 3rd party packages. The `from __future__ import unicode_literals` is special, as that must be before all other imports.

Different projects may also group certain 3rd party imports if they are paramount to the project. So, for example, in CMS projects, often there will be a block of django imports, then a block of cms imports, then all other 3rd party imports, then "local" imports... each block separated by a space.

Obviously, this is a matter of style, but its a style I support =)